apps/storybook/src/EventCard.stories.tsx

```diff
@@ -12,6 +12,7 @@ export default {
 
 const props = (): ComponentProps<typeof EventCard> => ({
   ...createEventResponse(),
+  title: text('Event Name', 'Example Event'),
```

Comment:
Don't think this is necessary since this behavior can already be tested in the lower level `EventInfo.stories`. Can you instead add a knob for the new `Title Character Limit` to `EventInfo.stories`? Then lgtm